py/specs/outer-joins.toml

```diff
@@ -10,20 +38,42 @@ statement_files = ["sql/employees.sql", "sql/offices.sql"]
         source = "data/offices.json"
 
 statements = [
-    "refresh table employees, offices"
+    "refresh table actor, film_actor, film, film_category, category, employees, offices"
 ]
 
 [[queries]]
-statement = "select e.name as employee, o.name as office from employees e left join offices o on e.office_id = e.id"
-iterations = 250
+statement = """SELECT 
+    film.film_id AS FID,
+    film.title AS title,
+    film.description AS description,
+    category.name AS category,
+    film.rental_rate AS price,
+    film.length AS length,
+    film.rating AS rating
+FROM 
+    category 
+    LEFT JOIN film_category ON category.category_id = film_category.category_id 
+    LEFT JOIN film ON film_category.film_id = film.film_id
+    JOIN film_actor ON film.film_id = film_actor.film_id
+    JOIN actor ON film_actor.actor_id = actor.actor_id;
+"""
+iterations = 500
 
 [[queries]]
-statement = "select e.name as employee, o.name as office from employees e right join offices o on e.office_id = e.id"
-iterations = 250
+statement = "select count(*) from (select e.name as employee, o.name as office from employees e right join offices o on e.office_id = e.id) t"
```

Comment:
I'm not sure if we want to mix count aggregation here and below.
If we provide a high LIMIT (=MxN records) isn't it sufficient to avoid fluctuations of the test time results?

Reply:
Problem with setting the high limit is that I ran into circuit breaker issues due to the large http request. We could think about filtering the data so that the resultset is smaller?